python/dgl/graphbolt/impl/sampled_subgraph_impl.py

```diff
@@ -68,6 +68,63 @@ def __post_init__(self):
             ), "Nodes in pairs should be of type torch.Tensor."
 
 
+def _to_reverse_ids(node_pair, reverse_row_node_ids, reverse_column_node_ids):
```

Comment:
` reverse_row_node_ids` is not the compacted counterpart of  `node_pair[0]`, same for `reverse_column_node_ids`. For example:

```
origin_node_pair = ([100, 50, 50, 100], ...)
after compact 
node_pair = ([1, 0, 0, 1], ...)
reverese_row_node_ids = [50, 100], means 50 -> 0 and 100 -> 1.
```

Reply:
This is also my understanding. The code below is to get the `origin_node_pair` of your example. It works because `reverse_row_node_ids[node_pair] = [100, 50, 50, 100]`.